traceapp/tmpl/data_vfsdata.go

```diff
@@ -0,0 +1,226 @@
+			uncompressedSize:  6514,
+		},
+	}
+
+	fs["/"].(*_vfsgen_dirInfo).entries = []os.FileInfo{
+		fs["/aggregate.html"].(os.FileInfo),
+		fs["/dashboard.html"].(os.FileInfo),
+		fs["/layout.html"].(os.FileInfo),
+		fs["/root.html"].(os.FileInfo),
+		fs["/trace.html"].(os.FileInfo),
+		fs["/traces.html"].(os.FileInfo),
+	}
+
+	return fs
+}()
+
+type _vfsgen_fs map[string]interface{}
+
+func (fs _vfsgen_fs) Open(path string) (http.File, error) {
+	// TODO: Maybe clean path?
```

Comment:
Good point, opened https://github.com/shurcooL/vfsgen/issues/10 and will aim to resolve it soon. After that, regenerating will remove this TODO comment.